src/test/java/com/bakdata/kafka/StreamsDeadLetterParserTest.java

```diff
@@ -144,15 +145,9 @@ private static Headers generateDefaultHeaders() {
 
     @Test
     void shouldConvert() {
-        final Headers headers = new RecordHeaders()
```

Comment:
I think this was intenionally verbose because the values are asserted here. The default headers method is used in other tests because we need them to parse the record but we don't assert them